PROJECT_IDEA_TEMPLATE.md

```diff
@@ -7,7 +7,9 @@
 - Description:
 - Expected Outcome:
 - Recommended Skills:
-- Mentor(s):
+- Mentor(s): # It is recommended to have at least 2 mentors, and at least one of them should be the primary mentor. For GSoC, it is **required** to have at least 2 mentors.
+  - Jane Doe (@jane-github, jane@email.address) - primary
+  - John Doe (@john-github, john@email.address)
```

Comment:
We should find new sample names -- Jane and John Doe are a bit macabre

Reply:
I can write
```
- Mentor Name
- Mentor Name
```

Any other suggestions?

Reply:
I was thinking we could take the opportunity to use more diverse names.